apps/meteor/app/apps/server/bridges/livechat.ts

```diff
@@ -135,6 +135,18 @@ export class AppLivechatBridge extends LivechatBridge {
 		return true;
 	}
 
+	protected async findRoomsByAgentId(agentId: string, appId: string): Promise<ILivechatRoom[]> {
```

Comment:
Since this is a find, can you create also a function that returns just the count? In case we need it

Reply:
performance wise, we prefer to do this on the DB instead of just using `length` right?

Reply:
Yup, less data to send to the app and to keep in mem